requirements.txt

```diff
@@ -1,3 +1,3 @@
 django-netjsongraph>=0.4.2,<0.5.0
-openwisp-users>=0.1.8,<0.2
+git+git://github.com/openwisp/openwisp-users.git#openwisp-users
```

Comment:
I don't know how to get latest openwisp-users

Reply:
no don't do this, load it in the travis build

Reply:
ok i'll try install from travis